fix: crash when updating import.http config (#2204)
grafana/alloy: CHANGELOG.md

- Updated `prometheus.write.queue` to fix issue with TTL comparing different scales of time. (@mattdurham)

> - Fixed a crash when updating import.http config that occurred because `importsource.HTTPArguments` was passed to `component/remote/http.Update` instead of `component/remote/http.Arguments`

Reviewer: ```suggestion
- Fixed a crash when updating the configuration of `remote.http`. (@kinolaev)
```
The crash seems to happen any time the config is updated, so we don't need to go in too much detail about when it happens.

If you would prefer not to mention your username, that's ok. But ideally we would like to give credit to contributions :)
